ddtrace/contrib/internal/litellm/patch.py

```diff
@@ -0,0 +1,94 @@
+from importlib.metadata import version
+import sys
+
+import litellm
+
+from ddtrace import config
+from ddtrace.contrib.internal.litellm.utils import tag_request
+from ddtrace.contrib.trace_utils import unwrap
+from ddtrace.contrib.trace_utils import with_traced_module
+from ddtrace.contrib.trace_utils import wrap
+from ddtrace.internal.utils import get_argument_value
+from ddtrace.llmobs._integrations import LiteLLMIntegration
+from ddtrace.trace import Pin
+
+
+config._add(
+    "litellm",
+    {},
+)
+
+
+def get_version() -> str:
+    try:
+        return version("litellm")
+    except Exception:
+        return ""
+
+
+def _create_span(litellm, pin, func, instance, args, kwargs):
+    integration = litellm._datadog_integration
+    model = get_argument_value(args, kwargs, 0, "model", None)
+    span = integration.trace(
+        pin,
+        "litellm.%s" % func.__name__,
```

Comment:
since the span name is already `litellm.request`, I don't think we need the `litellm.` prefix for the resource name.